geode-for-redis/src/distributedTest/java/org/apache/geode/redis/OutOfMemoryDUnitTest.java

```diff
@@ -181,7 +182,7 @@ public void shouldReturnOOMError_forWriteOperations_whenThresholdReached() {
 
     await()
         .untilAsserted(() -> assertThatThrownBy(() -> jedis.set(server1Tag + "oneMoreKey", "value"))
-            .hasMessageContaining("OOM"));
+            .hasMessage("OOM " + ERROR_OOM_COMMAND_NOT_ALLOWED));
```

Comment:
Does it make sense to pull these strings like "OOM", "ERR", "WRONGTYPE", etc. into constants? They're unlikely to change, of course, but they are repeated in multiple files and are part of the actual output.

Reply:
It might make sense to make "OOM", "ERR" and "WRONGTYPE" just part of the error string. We currently have multiple flavours of "getXResponse" methods in `Coder` which just tack a different string onto the beginning of the error message. We could simplify things a fair bit by baking that prefix into the error message.